src/main/sensors/gyro.c

```diff
@@ -81,9 +81,7 @@ FASTRAM gyro_t gyro; // gyro sensor object
 
 STATIC_UNIT_TESTED gyroDev_t gyroDev0;  // Not in FASTRAM since it may hold DMA buffers
 STATIC_FASTRAM int16_t gyroTemperature0;
-
-STATIC_FASTRAM_UNIT_TESTED zeroCalibrationVector_t gyroCalibration;
-STATIC_FASTRAM int32_t gyroADC[XYZ_AXIS_COUNT];
+STATIC_FASTRAM_UNIT_TESTED zeroCalibrationVector_t gyroCalibration0;
```

Comment:
Maybe this could be a 1 element array already?

Reply:
Done